packages/next/src/build/index.ts

```diff
@@ -2018,13 +2020,16 @@ export default async function build(
                       pagePath
                     )
 
+                const isInsideAppDir = pageType === 'app'
                 const staticInfo = pagePath
-                  ? await getPageStaticInfo({
+                  ? await getStaticInfoIncludingLayouts({
+                      isInsideAppDir,
                       pageFilePath,
-                      nextConfig: config,
-                      // TODO: fix type mismatch
```

Comment:
@timneutkens This TODO comment was added at [PR 60107](https://github.com/vercel/next.js/pull/60107/files#diff-90d1d5f446bdf243be25cc4ea2295a9c91508859d655e51d5ec4a3562d3a24d9R1584), are we safe to remove it?

I removed the comment since it is identical to what it is in the [getStaticInfoIncludingLayouts](https://github.com/vercel/next.js/blob/73c8d414ed44bae6382f34035d099a2199ee1705/packages/next/src/build/entries.ts#L122).